de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5484320_sys_gh3413_CreateTableAndWindo_C_BPartner_CreditLimit.sql

```diff
@@ -0,0 +1,1184 @@
+-- 2018-01-31T14:17:00.493
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Table (AccessLevel,ACTriggerLength,AD_Client_ID,AD_Org_ID,AD_Table_ID,CopyColumnsFromTable,Created,CreatedBy,EntityType,ImportTable,IsActive,IsAutocomplete,IsChangeLog,IsDeleteable,IsDLM,IsHighVolume,IsSecurityEnabled,IsView,LoadSeq,Name,ReplicationType,TableName,Updated,UpdatedBy) VALUES ('3',0,0,0,540929,'N',TO_TIMESTAMP('2018-01-31 14:17:00','YYYY-MM-DD HH24:MI:SS'),100,'D','N','Y','N','N','Y','N','N','N','N',0,'Bussines Partner Credit Limit','L','C_BPartner_CreditLimit',TO_TIMESTAMP('2018-01-31 14:17:00','YYYY-MM-DD HH24:MI:SS'),100)
+;
```

Comment:
if i see this correctly, you have `IsChangeLog=N` ..i recommend to have it =Y..this table is about a sensitive topic and even if this table is "append-only", it can't hurt

Reply:
Agree!